CONTRIBUTING.md

```diff
@@ -24,10 +23,14 @@ guidance below.
 
 ## Changing the codebase
 
-Generally speaking, you should fork this repository, make changes in your
-own fork, and then submit a pull-request. For timely code reviews,
-please tag @cfpb/cfgov-backends and @cfpb/cfgov-frontends as appropriate
-for your changes.
+If you are a contributor from outside of CFPB, you should fork this 
+repository, make changes in your own fork, and then submit a pull-request. 
```

Comment:
I see it in the original code, but since we're updating it: "pull request" has no hyphen.